src/api/cartAPI.ts

```diff
@@ -1,6 +1,10 @@
 import { getCookie } from '@/libs/manageCookie';
+import type { OptionChageAPIType } from '@/types/CartTypes';
+import { CustomKeyboardAPITypes } from '@/types/CustomKeyboardTypes';
 import { CartProductType } from '@/types/ProductTypes';
 
+const BASEURL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;
+
```

Comment:
```suggestion
const BASE_URL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;

```
사소하지만 언더바 있는게 더 좋을거같군요